torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -745,6 +745,19 @@ RegisterOperators reg(
            handler(ss.str());
          },
          aliasAnalysisSpecialCase()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA("prim::Concat(...) -> Tensor"),
```

Comment:
I think we probably want `prim::ConstantConcat`, following along the lines of `prim::ConstantChunk`. I'm not sure though.

Reply:
I'm open to changing that, if everyone prefers that.